### ISSUE TITLE
problem in install waymo

### ISSUE THREAD

**Issue by unssww** [huawei-noah/SMARTS#1636]
**High Level Description**
ERROR: Could not find a version that satisfies the requirement waymo-open-dataset-tf-2-4-0; extra == "waymo" (from smarts[waymo]) (from versions: none)
ERROR: No matching distribution found for waymo-open-dataset-tf-2-4-0; extra == "waymo"

**Desired SMARTS version**
0.6.1rc1

**Operating System**
 macOS Monterey 12.4

**Problems**
**1、when **make sanity-test** ,    there is    ModuleNotFoundError ,** 

(.venv) (base) songsong@songsongdeMacBook-Pro smarts % make sanity-test              
scl scenario build-all scenarios
Waiting on scenarios/zoo_intersection ...
build-scenario scenarios/intersections/4lane_t
build-scenario scenarios/zoo_intersection
build-scenario scenarios/intersections/2lane
build-scenario scenarios/intersections/4lane
build-scenario scenarios/straight
build-scenario scenarios/NGSIM/i80
Waiting on scenarios/intersections/4lane_t ...
Waiting on scenarios/intersections/4lane ...
(<class 'ModuleNotFoundError'>, ModuleNotFoundError("No module named 'waymo_open_dataset'"), <traceback object at 0x7fa91945f800>)
You may not have installed the [waymo] dependencies required to use the waymo replay simulation. Install them first using the command `pip install -e .[waymo]` at the source directory.

**2、so i install waymo,but still error.**

ERROR: Could not find a version that satisfies the requirement waymo-open-dataset-tf-2-4-0; extra == "waymo" (from smarts[waymo]) (from versions: none)
ERROR: No matching distribution found for waymo-open-dataset-tf-2-4-0; extra == "waymo"

**3， then i download waymo_open_dataset_tf_2_4_0-1.4.1-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
the file could not install in mac.**   .   

**4 version for mac  is not  provided . how can i install waymo in mac?**

**Comment by saulfield** [comment]
Hi @unssww,

For your first issue: waymo should not be required for running `make sanity-test`. I just confirmed this on the latest version of the `develop` branch, installing & running from the SMARTS repo.

For the other issues: if you are looking to use waymo, it appears they do not support macOS directly. From a quick skim of the waymo repo, it looks like it might be possible, see: https://github.com/waymo-research/waymo-open-dataset/issues/67

**Comment by saulfield** [comment]
Closing as this seems to not relate directly to SMARTS. Please feel free to reopen this or create a new issue if there is something we can help with on our end.
